src/openclpow.py

```diff
@@ -20,11 +20,12 @@
 hash_dt = None
 
 try:
-    import numpy
     import pyopencl as cl
-except:
+    import numpy
+except ImportError:
     libAvailable = False
```

Comment:
better log it or display it with help command.
print ("Please import numpy module : pip install numpy")

Reply:
@omkar1117 maybe something like "Please install numpy package if you want to use opencl PoW:\n pip install numpy"? It's useful only with AMD GPU, not every user needs opencl.

Reply:
Though it's already done in `bitmessagemain` by `depends.check_dependencies()` call.